src/main/java/utils/ExportResultsUtil.java

```diff
@@ -1,74 +1,55 @@
 package utils;
 
-import refactoring.MoveMethodRefactoring;
-
-import com.intellij.psi.PsiMethod;
-import com.intellij.psi.PsiClass;
+import refactoring.Refactoring;
+import javax.swing.*;
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.nio.file.StandardOpenOption;
 import java.util.List;
-import java.util.function.Function;
-import java.util.Optional;
-
 import static java.nio.file.Files.createFile;
 import static java.nio.file.Files.write;
-import static utils.PsiUtils.getHumanReadableName;
 
 /**
  * Class exports (MoveMethod) refactoring results to the file
```

Comment:
Please, fix this comment. Firstly, there is no need to write a word "Class". Secondly, this class is responsible to export  suggestions for all kinds of refactoring (Move method, Export method, etc).